src/Graphics/src/Graphics.Skia/SkiaImage.cs

```diff
@@ -100,38 +100,41 @@ public IImage Resize(float width, float height, ResizeMode resizeMode = ResizeMo
 
 		public void Save(Stream stream, ImageFormat format = ImageFormat.Png, float quality = 1)
 		{
-			throw new NotImplementedException();
-			// todo: implement me
+			var skStream = GetSkStream(format, quality);
 
-			/*
-		 switch (format)
-		 {
-			case ImageFormat.Jpeg:
-			   image.Compress (Bitmap.CompressFormat.Jpeg, (int)(quality * 100), stream);
-			   break;
-			default:
-			   image.Compress (Bitmap.CompressFormat.Png, 100, stream);
-			   break;
-		 }
-			*/
+			using (skStream)
+			{
+				skStream.CopyTo(stream);
+			}
 		}
 
-		public Task SaveAsync(Stream stream, ImageFormat format = ImageFormat.Png, float quality = 1)
+		public async Task SaveAsync(Stream stream, ImageFormat format = ImageFormat.Png, float quality = 1)
 		{
-			throw new NotImplementedException();
-			// todo: implement me
+			var skStream = GetSkStream(format, quality);
+			using (skStream)
+			{
+				await skStream.CopyToAsync(stream);
+			}
+		}
 
-			/*
-		 switch (format)
-		 {
-			case ImageFormat.Jpeg:
-			   await image.CompressAsync (Bitmap.CompressFormat.Jpeg, (int)(quality * 100), stream);
-			   break;
-			default:
-			   await image.CompressAsync (Bitmap.CompressFormat.Png, 100, stream);
-			   break;
-		 }
-			*/
+		private Stream GetSkStream(ImageFormat format, float quality)
+		{
+			// Skia quality range from 0-100, this is supported by jpeg and webp. Higher values correspond to improved visual quality, but less compression.
+			const int MaxSKQuality = 100; 
+			var skQuality = (int)(MaxSKQuality * quality);
+			SKEncodedImageFormat skEncodedImageFormat = format switch
+			{
+				ImageFormat.Png => SKEncodedImageFormat.Png,
+				ImageFormat.Jpeg => SKEncodedImageFormat.Jpeg,
+				ImageFormat.Bmp => SKEncodedImageFormat.Bmp,
+				ImageFormat.Gif => SKEncodedImageFormat.Gif,
```

Comment:
I am not so sure about bmp and gif... i do kniw skia has limited save options. Probably png, jpg and webp i think.

Reply:
Thank you @mattleibow  and I will test it.

Reply:
@lindexi did you get around to testing this?

Reply:
@jfversluis I am testing...

Reply:
@mattleibow and @jfversluis The skia do not support bmp and gif and the `Encode` method will return null when the `skEncodedImageFormat` is Bmp or Gif.

Reply:
@mattleibow and @jfversluis  Could you review my PR ?